src/api.ts

```diff
@@ -302,3 +303,48 @@ export const getTafsirContent = (
     }),
   );
 };
+
+/**
+ * Get reflections from QuranReflect API
+ *
+ * @param {string} chapterId
+ * @param {string} verseFrom
+ * @param {string} verseTo
+ * @returns {Promise}
+ */
+export const getQuranReflectData = (chapterId: string, verseFrom: string, verseTo: string) => {
```

Comment:
2 things:

1. `export` shouldn't be added since this is used locally.
2. I think a better naming would be `fetchQuranReflectVerseData` .